client/src/templates/Challenges/utils/polyvinyl.js

```diff
@@ -13,7 +13,7 @@ export function castToObservable(maybe) {
     return maybe;
   }
   if (isPromise(maybe)) {
-    return Observable.fromPromise(maybe);
+    return from(maybe);
```

Comment:
Nice catch, I figured this out is a different part of the codebase but forgot this one.